book/quick-brick/Setup.md

```diff
@@ -5,57 +5,151 @@
+Please refer to the [environment setup guide](/dev-env/node.md) to setup your node environment.
+
+To prepare a project for quick-brick you need to [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the official [QuickBrick](https://github.com/applicaster/QuickBrick) repository. This Repo contains the source code for the Quick Brick Project, which enables to create a Zapp App with a 100% react-native-flavored UI.
 
 ### Native
 
-QuickBrick relies on the ZappAppleBuilder & Zapp-Android apps to run on devices or on the emulator. This means that you need to have fully functionnal ZappAppleBuilder & Zapp-Android environment.
+QuickBrick relies on the [ZappAppleBuilder](https://github.com/applicaster/ZappAppleBuilder) & [Zapp-Android](https://github.com/applicaster/Zapp-Android) apps to run on devices or on the emulator. This means that you need to have fully functional ZappAppleBuilder & Zapp-Android environments. 
+
 This includes:
 
-- access to both ZappAppleBuilder & Zapp-Android repos
-- [ruby environment](/dev-env/ruby.md)
-- latest zapptool
-- Xcode 11.2.x
-- Android studio 3.1.1 with Gradle 4.5.1
+- Access to both [ZappAppleBuilder](https://github.com/applicaster/ZappAppleBuilder) & [Zapp-Android](https://github.com/applicaster/Zapp-Android) repositories.
+- Align with our [Ruby environment](/dev-env/ruby.md) requirements. 
+- Use the latest [Zapptool](https://github.com/applicaster/ZappTool)
+- Xcode 11.2.1
+- Android studio 3.5.3 with Gradle 4.5.1
+
+Please clone the [ZappAppleBuilder](https://github.com/applicaster/ZappAppleBuilder) & [Zapp-Android](https://github.com/applicaster/Zapp-Android) repositories. Note that they don't need to be in the same folder. They can be located anywhere on your environment.
+
+
+## Development Workflow
 
-The ZappAppleBuilder and Zapp-Android workspaces don't need to be in the same folder. They can be located anywhere on your environment.
+Before development, make sure you have a working iOS or Android application on the [Zapp](https://zapp.applicaster.com/) website.
 
-## Project workflow
+In case you don't have a working iOS or Android application, create one for your platforms. You can use any recent SDK. We recommend to use the latest stable on each platform.
 
-First thing you need is a Zapp app. Create one version for iOS and one for android. You can use any recent SDK, but for a start we recommend the latest dev on each platform.
-Pick up your your app version ids and save them for later.
+Copy your app version ids and save them for later usage. Note that each version has its unique app id.
 
-### Prepare your QuickBrick workspace (when contributing on the main repository)
+### Prepare a QuickBrick workspace
+
+Go to the [forked](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) [QuickBrick](https://github.com/applicaster/QuickBrick) repository directory and run `yarn setup` to automatically install all dependencies.
```

Comment:
Yarn setup was only meant to load flow type definitions when working on the core repo. Running `yarn` alone is enough to install dependencies

Reply:
Like I have written above, this file is an old file. 
I have deleted it. :)